### DISCUSSION TITLE
How run AT from manual test-cases?

### DISCUSSION THREAD

**Discussion by Chromegolf** [allure-framework/allure2#2420] [TestOps]
Hello
We have some manual test-cases and relevant him AT. We want run AT from relevant manual test-cases to check it and compare scenario on review.
How we can associate manual test-cases and AT for our problem and run AT from selected manual test-cases?
Please give docs or settings example.
Thx

**Comment by cheshi-mantu** [comment]
Hi @Chromegolf 
you need to add AllureId to your automated test cases
check here https://www.youtube.com/watch?v=CMZMmkuczig&list=PLINOSjry419nVTsmiQ4ACukWDioU4QK2R&index=6
or here: https://docs.qameta.io/allure-testops/ecosystem/intellij-plugin/#manual-test-cases-automation

if you would shed more light on the test framework used, I could point to the docs related to the integration with allure framework on our site https://allurereport.org/
